src/store/middlewares/peers.js

```diff
@@ -1,25 +1,5 @@
-import { activePeerSet, activePeerUpdate } from '../../actions/peers';
-import actionTypes from '../../constants/actions';
-import networks from './../../constants/networks';
-import getNetwork from './../../utils/getNetwork';
-import localJSONStorage from './../../utils/localJSONStorage';
-
-const peersMiddleware = store => next => (action) => {
+const peersMiddleware = () => next => (action) => {
   next(action);
-
-  const network = Object.assign({}, getNetwork(networks.default.code));
-  const hasNoSavedAccounts = !localJSONStorage.get('accounts', []).length;
-
-  switch (action.type) {
-    case actionTypes.storeCreated:
-      /* istanbul ignore else  */
-      if (hasNoSavedAccounts) {
-        store.dispatch(activePeerSet({ network, noSavedAccounts: true }));
-        store.dispatch(activePeerUpdate({ online: true }));
```

Comment:
I had to dispatch `storeCreated` as I didn't found any place where that was called. 
that's why I thought removing it will not make any effect. 
```
store.dispatch({ data: {}, type: actionTypes.storeCreated });
```